Lib/test/test_sqlite3/test_dbapi.py

```diff
@@ -524,6 +539,13 @@ def __getitem__(slf, x):
         with self.assertRaises(ZeroDivisionError):
             self.cu.execute("select name from test where name=?", L())
 
+    def test_execute_too_many_params(self):
+        category = sqlite.SQLITE_LIMIT_VARIABLE_NUMBER
+        msg = "too many SQL variables"
+        with cx_limit(self.cx, category=category, limit=1):
```

Comment:
I would set the limit to 2 and test that the request with 2 variables is accepted, but with 3 variables is not.

And test also with named variables.

Reply:
SGTM, I'll add that.